pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -346,9 +347,21 @@ async def run(self):
         if self.release_version[0] < 4:
             _LOGGER.info("Don't verify payloads for OCP3 releases")
         else:
-            _LOGGER.info("Verify the swept builds match the nightlies...")
-            for _, payload in self.candidate_nightlies.items():
-                self.verify_payload(payload, advisories["image"])
+            _LOGGER.info("Verify the swept builds match the imagestreams that were updated during build-sync...")
+            self.image_stream_x_value = self.release_version[0]
+            self.image_stream_y_value = self.release_version[1]
+            self.image_stream_version = f'{self.image_stream_x_value}.{self.image_stream_y_value}'
```

Comment:
```suggestion
            image_stream_version = f'{self.release_version[0]}.{self.release_version[1]}'
```

Reply:
concise, and also we don't need a class var `self.image_stream_version`, local var `image_stream_version` is fine here